src/card.tsx

```diff
@@ -726,3 +732,199 @@ export const DisplayDataCard = React.forwardRef<HTMLDivElement, DisplayDataCardP
         />
     )
 );
+
+interface PosterCardBaseProps {
+    backgroundImage: string;
+    ariaLabel?: string;
+    aspectRatio?: AspectRatio | number;
+    width?: number | string;
+    height?: number | string;
+    icon?: React.ReactElement;
+    actions?: Array<CardAction>;
+    onClose?: () => void;
+    dataAttributes?: DataAttributes;
+
+    headline?: string | RendersNullableElement<typeof Tag>;
+    pretitle?: string;
+    pretitleLinesMax?: number;
+    title?: string;
+    titleLinesMax?: number;
+    description?: string;
+    descriptionLinesMax?: number;
+}
+
+interface PosterCardToProps extends PosterCardBaseProps {
+    to?: string;
+    fullPageOnWebView?: boolean;
+}
+
+interface PosterCardHrefProps extends PosterCardBaseProps {
+    href?: string;
+    newTab?: boolean;
+}
+
+interface PosterCardOnPressProps extends PosterCardBaseProps {
+    onPress?: () => void;
+}
+
+type PosterCardProps = ExclusifyUnion<PosterCardToProps | PosterCardHrefProps | PosterCardOnPressProps>;
+
+const getPosterTouchableProps = (props: {
+    onPress?: () => void;
+    href?: string;
+    newTab?: boolean;
+    to?: string;
+    fullPageOnWebView?: boolean;
+}) => {
+    if (props.onPress) return {onPress: props?.onPress};
+
+    if (props.href) {
+        return {
+            newTab: props.newTab,
+            href: props.href,
+        };
+    }
+    if (props.to) {
+        return {
+            to: props.to,
+            fullPageOnWebView: props.fullPageOnWebView,
+        };
+    }
+
+    return undefined;
+};
+
+const POSTER_CARD_MIN_WIDTH = 140;
+const POSTER_CARD_MIN_HEIGHT = 112;
```

Comment:
do you have a best practice for this? I didn't want to place it as a default argument in order to avoid magic numbers